lib/target/ti83p/classic/ti83papp_crt0.asm

```diff
@@ -0,0 +1,268 @@
+;	Stub for the TI 83+ calculator for building as an app
+;
+;  This is perferable to ti83papp.asm
+;
+;
+	MODULE  Ti83plus_App_crt0
+
+	DEFINE TI83PLUSAPP	;Used by grayscale interrupt and the such
+
+
+	EXTERN	_main		; No matter what set up we have, main is
+				;  always, always external to this file.
+	
+
+
+
+
+;-------------------------
+; Begin of (shell) headers
+;-------------------------
+
+	INCLUDE "Ti83p.def"	; ROM / RAM adresses on Ti83+[SE]
+        defc    crt0 = 1
+	INCLUDE	"zcc_opt.def"	; Receive all compiler-defines
+
+	defc	CONSOLE_ROWS = 8
+        defc    TAR__clib_exit_stack_size = 3
+        defc    TAR__register_sp = -1
+	defc	__CPU_CLOCK = 6000000
+        INCLUDE "crt/classic/crt_rules.inc"
+	
+
+
+; Header data
+	DEFINE ASM
+	DEFINE NOT_DEFAULT_SHELL
+
+	org $4000
```

Comment:
Is `CRT_ORG_BSS` needed to place variables into an area of RAM?

Reply:
I think this might also be the reason why `puts()` is failing - the std* streams aren't being setup because they're in flash rather than ram